MREUnityRuntime/MREUnityRuntimeLib/Core/Physics/PredictionInterpolation.cs

```diff
@@ -354,5 +354,11 @@ public void StartBodyPredicitonForNextFrame()
 			} // end for each 
 
 		} // end of PredictAllRemoteBodiesWithOwnedBodies
+
+		public void Clear()
+		{
```

Comment:
I am not sure why are these Clears needed, they should be cleared in each step.

Reply:
But it is fine with me to have this in.

Reply:
to clean up storage when client disconnects

---
In reply to: [448321029](https://github.com/microsoft/mixed-reality-extension-unity/pull/199#discussion_r448321029) [](ancestors = 448321029)